client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,5 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
```

Comment:
Is there any use of having timestamps on condensed block if it's going to be hidden with CSS anyway?